static/modes/ptx-mode.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2018, Compiler Explorer Authors
```

Comment:
We're in 2019!

Reply:
Blimey, time really flies by :slightly_smiling_face: